src/Http/Http.Abstractions/src/Routing/RouteValueDictionary.cs

```diff
@@ -95,6 +95,7 @@ public RouteValueDictionary()
     /// property names are keys, and property values are the values, and copied into the dictionary.
     /// Only public instance non-index properties are considered.
     /// </remarks>
+    [RequiresUnreferencedCode("RouteValueDictionary uses reflection to generate the route values.")]
```

Comment:
```suggestion
```
Will look at this change again in followup PR to resolve Http.Abstractions warnings